<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkTimeClass</class>
 <widget class="QMainWindow" name="WorkTimeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WorkTime</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="settings">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/WorkTime/C:/Users/82436/Downloads/images/cogs-icon.png</normaloff>:/WorkTime/C:/Users/82436/Downloads/images/cogs-icon.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="top">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/WorkTime/C:/Users/82436/Downloads/images/maximize.png</normaloff>:/WorkTime/C:/Users/82436/Downloads/images/maximize.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="small">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/WorkTime/C:/Users/82436/Downloads/images/minimize_window.png</normaloff>:/WorkTime/C:/Users/82436/Downloads/images/minimize_window.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closewindow">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/WorkTime/C:/Users/82436/Downloads/images/close_window.png</normaloff>:/WorkTime/C:/Users/82436/Downloads/images/close_window.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Time</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="Tips">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Come on ! Time is passing !</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLCDNumber" name="lcdNumber">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="digitCount">
                 <number>8</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="allTime">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="smallDecimalPoint">
                 <bool>false</bool>
                </property>
                <property name="digitCount">
                 <number>7</number>
                </property>
                <property name="mode">
                 <enum>QLCDNumber::Dec</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Filled</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="startWork">
                <property name="styleSheet">
                 <string notr="true">color: rgb(85, 255, 127);</string>
                </property>
                <property name="text">
                 <string>开始工作</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stopWork">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 0, 0);</string>
                </property>
                <property name="text">
                 <string>停止工作</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>When</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QListWidget" name="listWidget"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="text">
                     <string>开始日期</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDateEdit" name="dateEdit"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>截止日期</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDateEdit" name="dateEdit_2"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QProgressBar" name="progressBar">
                  <property name="value">
                   <number>36</number>
                  </property>
                  <property name="format">
                   <string>%p%</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WorkTime.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startWork</sender>
   <signal>clicked()</signal>
   <receiver>WorkTimeClass</receiver>
   <slot>StartWork()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopWork</sender>
   <signal>clicked()</signal>
   <receiver>WorkTimeClass</receiver>
   <slot>StopWork()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>top</sender>
   <signal>clicked()</signal>
   <receiver>WorkTimeClass</receiver>
   <slot>TopWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>small</sender>
   <signal>clicked()</signal>
   <receiver>WorkTimeClass</receiver>
   <slot>SmallWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closewindow</sender>
   <signal>clicked()</signal>
   <receiver>WorkTimeClass</receiver>
   <slot>CloseWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>StartWork()</slot>
  <slot>StopWork()</slot>
  <slot>TopWindow()</slot>
  <slot>SmallWindow()</slot>
  <slot>CloseWindow()</slot>
  <slot>changeProgress()</slot>
  <slot>settingPage()</slot>
  <slot>slot3()</slot>
  <slot>slot2()</slot>
  <slot>slot4()</slot>
 </slots>
</ui>
